x-pack/platform/plugins/shared/alerting/server/application/rule/methods/get_rule_ids_with_gaps/schemas/get_rules_with_gaps.ts

```diff
@@ -6,11 +6,29 @@
  */
 
 import { schema } from '@kbn/config-schema';
+import { gapFillStatus, gapStatus } from '../../../../../../common';
 
 export const getRuleIdsWithGapsParamsSchema = schema.object({
-  start: schema.string(),
-  end: schema.string(),
-  statuses: schema.maybe(schema.arrayOf(schema.string())),
+  start: schema.maybe(schema.string()),
+  end: schema.maybe(schema.string()),
+  statuses: schema.maybe(
+    schema.arrayOf(
+      schema.oneOf([
+        schema.literal(gapStatus.UNFILLED),
+        schema.literal(gapStatus.PARTIALLY_FILLED),
+        schema.literal(gapStatus.FILLED),
+      ])
+    )
+  ),
+  highestPriorityGapFillStatuses: schema.maybe(
+    schema.arrayOf(
+      schema.oneOf([
+        schema.literal(gapFillStatus.UNFILLED),
+        schema.literal(gapFillStatus.IN_PROGRESS),
+        schema.literal(gapFillStatus.FILLED),
+      ])
+    )
```

Comment:
nit: Please add comments for each of the params to elaborate on their usage and what each of them affect.